readme.md

```diff
@@ -3,7 +3,7 @@
 ## Kore
 
 Kore projects are built using koremake, a nodejs tool.
-In your project's directory call `node Kore/make` - this will
+In your project's directory call `node Kore/make --init` this will

```

Comment:
This should be reverted, the `--init` doesn't create the project files (makefile / vcproj / ...) to actually build the project, it only creates an initial `korefile.js` and `main.cpp`.